src/common/components/contactInfo/contactInfo.tpl.html

```diff
@@ -192,7 +192,7 @@ <h4 class="panel-title  border-bottom-small  visible" translate>Your Information
     <div class="mb">
       <h4 class="panel-title  border-bottom-small  visible" translate>Mailing Address</h4>
 
-      <address-form address="$ctrl.donorDetails.mailingAddress" parent-form="$ctrl.detailsForm"></address-form>
+      <address-form address="$ctrl.donorDetails.mailingAddress" parent-form="$ctrl.detailsForm" disabled="$ctrl.donorDetails.staff"></address-form>
```

Comment:
No it's a component. I added the ng-disabled attributes inside. I suppose I could use the ng-disabled name but it's not native angular functionality...